src/cross_toml.rs

```diff
@@ -87,24 +165,14 @@ impl CrossToml {
         self.get_bool(target, |b| b.build_std, |t| t.build_std)
     }
 
-    /// Returns the list of environment variables to pass through for `build`,
-    pub fn env_passthrough_build(&self) -> &[String] {
-        &self.build.env.passthrough
+    /// Returns the list of environment variables to pass through for `build` and `target`
+    pub fn env_passthrough(&self, target: &Target) -> (&[String], &[String]) {
+        self.get_vec(target, |_| &[], |t| &t.env.passthrough)
```

Comment:
This will need to be updated due to #801 for the new optional arrays, and can just use `|_| None`.